compose.go

```diff
@@ -53,6 +53,20 @@ func (cs *ComposeStore[V]) SetEx(ctx context.Context, key string, value V, ttl t
 	return nil
 }
 
+func (cs *ComposeStore[V]) GetEx(ctx context.Context, key string) (out V, ttl time.Duration, exists bool, err error) {
+	for _, s := range cs.stores {
+		out, ttl, exists, err = s.GetEx(ctx, key)
```

Comment:
```suggestion
func (cs *ComposeStore[V]) GetEx(ctx context.Context, key string) (V, time.Duration, bool, error) {
	var out V
	var tll time.Duration
	for _, s := range cs.stores {
		out, ttl, exists, err = s.GetEx(ctx, key)
```

Reply:
I don't see a difference besides two extra lines?

Reply:
In whole repo i didn't found any function with named return values. Let's keep the code consistent and move the initialization of variables in body of function.
Sorry i should have wrote the comment to it right away.

Reply:
I don't know, but in this case the return named parameters feel more readable to me :man_shrugging:

Reply:
OK ok :)